Makefile

```diff
@@ -68,7 +68,7 @@ endif
 LOCAL_VOLUME_ARGS   := -v$(CURDIR):/src:delegated -v $(GOPATH):/go:delegated
 GOPATH_WD_OVERRIDES := -w /src -e GOPATH=/go
 IMAGE_BUILD_FLAGS   := -e CGO_ENABLED=0 -e GOOS=linux -e GOARCH=${GOARCH}
-IMAGE_BUILD_ARGS     = --build-arg LABEL_VERSION=$(TAG) --build-arg LABEL_RELEASE=$(TAG) --build-arg QUAY_TAG_EXPIRATION=$(QUAY_TAG_EXPIRATION)
+IMAGE_BUILD_ARGS     = --build-arg LABEL_VERSION=$(TAG) --build-arg LABEL_RELEASE=$(TAG) --build-arg QUAY_TAG_EXPIRATION=$(QUAY_TAG_EXPIRATION) $(DB_DOCKERBUILD_ARGS)
```

Comment:
@kcrane just curious about this ^. Once resolved should be good to go

Reply:
I believe this will also make the Scanner image for Z based on UBI 9 instead of UBI 8, though, right? Originally, we just wanted this for Scanner DB.

I'm wondering if we should just create a `db-image-build-args` target which is:

```
.PHONY: db-image-build-args
db-image-build-args:
	@echo $(IMAGE_BUILD_ARGS) $(DB_DOCKERBUILD_ARGS)
```

and then revert the other changes to this file.

Reply:
Yes, you're right.  BASE_IMAGE is common across a lot of files.  Another option would be to rename BASE_IMAGE to DB_BASE_IMAGE in the Dockerfile in `image/db/rhel/Dockerfile*`.  This lets us keep the one set of image build arguments across all builds in the project.

Let me know and I will make the change.

Reply:
Since this repo is mostly in maintenance mode, whatever is easiest works for me